### DISCUSSION TITLE
My website looks different than my local host

### DISCUSSION THREAD

**Discussion by sytamdoan** [community/community#64555] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://github.com/community/community/assets/48368269/bec6936e-8ee6-4261-96f7-bd10988d2d25)
This is my local host
![image](https://github.com/community/community/assets/48368269/f1896fef-ba1e-40e8-8b26-3b99eeb40436)
This is what shows up on my github website

My repository: https://github.com/sytamdoan/sytamdoan.github.io

Does anyone know why it looks so different? Anything would help. Thank you.

**Comment by WilliamHCarter** [comment]
Hi! So the main things that stick out here are the image background color and the font. 

**Image Transparency**
For the image, a simple solution here would be to set the background color to something transparent, for example:
```
img{
    background-color: #fff0;
}
```
**Font Linking**
The second is the font. While you mention the font in your CSS file, you don't actually link to a font resource. This means that while this renders correctly on your local device where the font was installed, it won't render on the hosted machine which likely doesn't have times new roman installed. To fix this, you could try using [@font-face](https://www.w3schools.com/css/css3_fonts.asp) in your CSS, and link directly to a font file.

Hope this helps!

_if this answer was satisfactory, feel free to 'mark as answered' in the feed to help other users with similar questions_

**Reply by WilliamHCarter** [reply]
A few more tips:

**Doctype HTML**
Simply remove this line from your file, It allows for some consistency across legacy browsers, but considering that is it providing undefined behaviour, and that you are serving this from an MD file, its best to simply remove this

**Whitespace in READMO file**
On a number of lines in your readme file (29, 41, 43, 53, 57 etc. etc.) you have a blank line. this prevents the file from rendering correctly on your github repositroy page. I would recommend you remove these. If you want to create more whitespace, use the `<br>` tag to add it.

**Reply by sytamdoan** [reply]
Hi!
Thanks for the tip! I've been able to follow your advice and clean up alot of my code. Unfortunately I still wasn't able to make the background of the image transparent. I also tried using @font-face but my fonts remain unchanged, despite the fact that I added a .tff file and linked it in my CSS file. Do you have any advice?
Thank you for your help!

**Reply by WilliamHCarter** [reply, marked as answer]
Hi,
I'm glad that some of those comments were helpful, lets wrap up the last few issues:

**Image Transparency**
For this issue, it seems that there are some markdown-related styles getting in the way of our CSS. to fix this we can do the following:

In the html:
```
<img src="GitHub.png" alt="Github Page" id="image">
```

In the CSS:
```
#image {
    background-color: #fff0;
}
```

This uses the 'id' property to ensure that the specific image has a style applied by linking it to an ID.

**Font**
Font has a similar issue, where github default styles for the markdown format are more specific than our styles, we can fix this by replacing our `body{}` styles with the following:
```
.markdown-body {
    font-family: "Times_New_Roman";
    line-height: 1.6;
    max-width: 900px;
    margin: 0 auto;
    padding: 20px;
}
```

I hope this helps solve the remaining problems!

_if this answer was satisfactory, feel free to 'mark as answered' in the feed to help other users with similar questions_
